### PR TITLE
GDB-6280 Added verification for invalid links value on change. 

### PR DESCRIPTION
**Pull request by sava-savov-ontotext**
If such is detected proper error message is shown to the user. Added cypress test in visual.graph.spec verifying the change.

### CODE REVIEW

**Comment by sava-savov-ontotext** [comment]
Jenkins retry build

**Comment by sava-savov-ontotext** [comment]
Jenkins retry build

File: src/js/angular/graphexplore/controllers/graphs-visualizations.controller.js [hunk 1]
```diff
@@ -36,6 +37,10 @@ function GraphsVisualizationsCtrl($scope, $rootScope, $repositories, $licenseSer
     $scope.embedded = $location.search().embedded;
     $scope.openedNodeInfoPanel = undefined;
 
+    $scope.invalidLimit = false;
+    $scope.INVALID_LINKS_MSG = 'Invalid links limit';
+    $scope.INVALID_LINKS_TOOLTIP = 'The valid limit range is from 1 - 1000';
```
Comment (desislava-hristova-ontotext) [inline src/js/angular/graphexplore/controllers/graphs-visualizations.controller.js:42]: The message better be "The valid limit range is 1-1000" without the "from" or "valid limit range: 1-1000"

**Review by desislava-hristova-ontotext — APPROVED** [review]

**Comment by desislava-hristova-ontotext** [comment]
Jenkins retry build

**Comment by desislava-hristova-ontotext** [comment]
Jenkins retry build

**Pull request merged**